apps/site/lib/site_web/templates/layout/_new_nav_desktop.html.eex

```diff
@@ -20,6 +20,18 @@
               </div>
               <div class="m-menu-desktop__submenu-section" aria-label="<%= section_name %>">
                 <%= Enum.map(links, &SiteWeb.LayoutView.render_nav_link/1) %>
+                <%= if section_name == "Transit Police" do %>
+                  <div class="m-menu__feature emergency-contacts desktop">
+                    <div class="m-menu__section-heading">Emergency Contacts</div>
+                    <small>24 hours, 7 days a week</small>
+                    <div class="m-menu__feature_phone">
+                      <strong>Transit Police:</strong> <%= tel_link "617-222-1212" %>
+                    </div>
+                    <div class="m-menu__feature_phone">
+                      <strong>TTY:</strong> <%= tel_link "617-222-1200" %>
+                    </div>
+                  </div>
+                <%end %>
```

Comment:
Ah, good move! ➡️ 

I notice a missed opportunity to extract this section (& the identical section in the footer) into its own file, but maybe we can do that if we put this info in a third place.